osmosis/assetlist.json

```diff
@@ -14261,6 +14261,26 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/bOSMO.png"
         }
       ]
+    },
+    {
+      "description": "Memecoin for The International Brane Wave",
+      "denom_units": [
+        {
+          "denom": "factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz",
+          "exponent": 0
+        },
+        {
+          "denom": "brnz",
+          "exponent": 0
```

Comment:
Please do not define two separate denom units with the same exponent value (in this case 0). Instead, please add 'brnz' as an alias of factory/osmo13gu58hzw3e9aqpj25h67m7snwcjuccd7v4p55w/brnz.